### PR TITLE
Store all attributes changes when a feature is edited until it is saved

### PR DESCRIPTION
**Pull request by xavijam**
It will fix https://github.com/CartoDB/cartodb/issues/11010. Basically we store all attributes changed when it is edited before saving it. After that, changes history should be empty. 

**STTest**
1. Create a map of lines|points|polygons (take a dataset with several columns with data please).
2. Try to edit a feature.
3. Change the geometry (move the point, or any vertex of the line|polygon).
4. Edit any field of the feature form.
5. Save the changes.
6. Map should be refreshed.

cc @matallo

### CODE REVIEW

**Comment by matallo** [comment]
👏 👏 👏

File: lib/assets/javascripts/cartodb3/data/feature-definition-model.js [hunk 1]
```diff
@@ -125,10 +144,12 @@ var FeatureDefinitionModel = Backbone.Model.extend({
 
   fetch: function (options) {
     options = options || {};
+    this.off('change', null, null);
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/data/feature-definition-model.js]: Why the `null`, `null` params in the `this.off` function? Wouldn't it be better to write `this.off('change', this._onChange, this);`?

Comment (javierarce) [inline lib/assets/javascripts/cartodb3/data/feature-definition-model.js]: 💃

**Comment by xavijam** [comment]
Don't remove the branch when it is done.

**Pull request merged**